abstractions/java/lib/src/main/java/com/microsoft/kiota/serialization/ParseNode.java

```diff
@@ -45,6 +45,12 @@
      */
     @Nonnull
     Float getFloatValue();
+    /**
+     * Gets the Double value of the node.
+     * @return the Double value of the node.
+     */
+    @Nonnull
+    Double getDoubleValue();
```

Comment:
Looks like in java we can call `getFloatValue()` and `getDoubleValue()` while .Net only has `getFloatValue()`

Reply:
java was missing both methods, but dotnet already [had the double one](https://github.com/microsoft/kiota/blob/bugfix/identifiers-with-dots/abstractions/dotnet/src/serialization/IParseNode.cs#L50)